package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "action-create-release-pr",
+  "version": "0.0.16",
+  "description": "A GitHub Action for creating a release PR. Monorepo-compatible.",
+  "files": [
+    "lib/"
+  ],
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=12.10.0",
+    "yarn": ">=1.22.0 <2.0.0"
+  },
+  "scripts": {
+    "lint:eslint": "yarn eslint . --cache --ext js,ts",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '**/*.yml' --single-quote --ignore-path .gitignore",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "build:clean": "mkdir -p lib && rm -rf lib/* && mkdir -p dist && rm -rf dist/*",
+    "build:tsc": "tsc --project .",
+    "build:ncc": "ncc build lib/index.js --out dist",
+    "build": "yarn build:clean && yarn build:tsc && yarn build:ncc",
```

Comment:
In a later PR, we should make `build` a "non-clean" build, to allow incremental compilation. Maybe omit the `build:ncc` for that too? :thinking: If it's just for local dev.
Also we should add a `prepublishOnly` script to ensure the build runs.